doc/pages/upgrade/020-upgrade_011_10.md

```diff
@@ -0,0 +1,139 @@
+# Upgrade Operator v0.11-v1.0
+
+The upgrade procedure from v0.11 to v1.0 requires some manual intervention as the deployment and
+handling of the Operator's lifecylcle has changed: if v0.11 is entirely handled with `astartectl`,
+v1.0 employs Helm charts.
+
+Helm is intended to be used as the operator's lifecycle management tool, thus make sure you are
+ready with a working [Helm installation](https://helm.sh/docs/intro/install/).
+
+## Remove Astarte Operator v0.11
+
++ Remove the Operator's Service Account, Cluster Roles and Cluster Role Bindings:
+```bash
+kubectl delete serviceaccounts -n kube-system astarte-operator
+kubectl delete clusterroles.rbac.authorization.k8s.io astarte-operator
+kubectl delete clusterrolebindings.rbac.authorization.k8s.io astarte-operator
+```
+
++ Delete the Operator's deployment:
+```bash
+kubectl delete deployments.app -n kube-system astarte-operator
+```
+
+**DO NOT** delete Astarte's CRDs! This will lead to the deletion of the entire Astarte deployment
+with a consequent data loss.
+
+## Install cert-manager
+
+Please, before proceeding to the next steps make sure to be compliant with the new requirements for
+`v1.0`, i.e. if `cert-manager` is not installed yet, run the following commands:
+```bash
+$ helm repo add jetstack https://charts.jetstack.io
+$ helm repo update
+$ kubectl create namespace cert-manager
+$ helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --version v1.1.0 \
+  --set installCRDs=true
+```
+
+## Add Astarte Operator's Helm Chart Repository
+
+To restore the Operator's functionalities with v1.0, the first step is adding the Astarte Operator's
+Helm chart repository:
+```bash
+helm repo add astarte https://helm.astarte-platform.org
+helm repo update
+```
+
+## Prepare the Cluster to the Operator v1.0 Installation
+
+Set some environment variables that will come handy for the upcoming migration procedure:
++ `ASTARTE_OP_TEMPLATE_DIR` is the target directory in which the chart templates will be generated,
++ `ASTARTE_OP_RELEASE_NAME` is the name of the Astarte Operator deployment,
++ `ASTARTE_OP_RELEASE_NAMESPACE` is the namespace in which the Astarte Operator will reside.
+
+You can change the following values at your preference. However, the following values should be ok
+for the large majority of use cases.
+
+Note that you are responsible for checking that `ASTARTE_OP_RELEASE_NAMESPACE` exists within the
+cluster.
```

Comment:
I'd include the namespace creation in the instructions to maximise the copy-paste experience, if someone is coming from 0.11 they won't have the `astarte-operator` namespace so the normal path is creating it